taxcalc/policy_current_law.json

```diff
@@ -133,7 +133,7 @@
         "indexable": true,
         "indexed": false,
         "vi_name": "MARS",
-        "vi_vals": ["single", "joint", "separate", "headhousehold", "widow"],
+        "vi_vals": ["single", "mjoint", "marsep", "headhh", "widow"],
```

Comment:
@martinholmer These names are displayed over the parameter boxes on the webapp:
![Screen Shot 2019-04-12 at 11 38 54 AM](https://user-images.githubusercontent.com/9206065/56049509-9fe92b00-5d17-11e9-96da-40ffb5261cd0.png)

What do you think about choosing names that are more human readable like:

`["single", "married joint", "married separate", "head household", "widow"]`

The difference is only a few characters, and they are readable to those who are not familiar with the naming scheme.

Reply:
I think the new values for the vector indexes are more succinct and perfectly understandable to those using Tax-Calculator (or Tax-Brain) via Python programming or the `tc` command-line tool.  If you think  users of the TaxBrain webapp need longer labels on the boxes, then by all means you should supply them with longer box labels.  That's your call.